packages/gestalt/src/Popover.js

```diff
@@ -12,6 +16,10 @@ type Props = {|
    * Unique label to describe each Popover. Used for [accessibility](https://gestalt.pinterest.systems/web/popover#ARIA-attributes) purposes.
    */
   accessibilityLabel?: string,
+  /**
+   * Describe's the dismiss button's purpose. Default is "Close popover". Must be translated.
```

Comment:
We should be consistent on how we describe this pros. I'd prioritize `See the [dismiss button](https://gestalt.pinterest.systems/web/popover#Dismiss-button) variant to learn more.`

Default is "Close popover". >> too much info here? 

Must be translated. >> Must be **localized**?